include/unifex/config.hpp.in

```diff
@@ -228,7 +228,7 @@
 #elif defined(__GNUC__) && __GNUC__ >= 6
 #define UNIFEX_IS_SAME(...) __is_same_as(__VA_ARGS__)
 #elif UNIFEX_CXX_TRAIT_VARIABLE_TEMPLATES
-#define UNIFEX_IS_SAME(...) std::is_same_v<__VA_ARGS__>
+#define UNIFEX_IS_SAME(...) std::std::is_same_v<__VA_ARGS__>
```

Comment:
Perhaps we should delete `UNIFEX_CXX_TRAIT_VARIABLE_TEMPLATES` instead of updating what happens when it's set.

Reply:
Is there a downside to removing these macros?